contributing_to_docs/doc_guidelines.adoc

```diff
@@ -746,18 +748,36 @@ There might be advanced networking examples that require specific IP addresses,
 == Links, hyperlinks, and cross references
 Links can be used to cross-reference internal assemblies or send readers to external information resources for further reading.
 
-In OpenShift docs:
+In OpenShift Container Platform docs:
 
-* All links to internal content is created using `xref` and **must have an anchor ID**.
+* All links to internal content is created using `xref` and each `xref` **must have an anchor ID**.
 * Only use `xref` in assemblies, not in modules.
-* All links to external websites are created using `link`.
-* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
+* All links to external websites are created by using `link`.
+* Links between different distributions, or _distros_, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and must be in distro-specific files or a conditionalized state to apply to the relevant distros.
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+To maintain consistency across OpenShift Container Platform documentation, ensure that you adhere to the following guidance for lead-in link sentences:
+
+* Maintain the `For more information, see "<section_name>."` format when pointing to a section in the same document.
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+* When pointing to a link that exists in another customer document, use the `See "<section_name>" in _<document_name>_.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
+* You can add link sentences to either the _Additional resources_ section or the _Next steps_ section, but know the following differences:
```

Comment:
I'm not sure if this is accurate. I thought that the mod docs template didn't want us to use full sentences in the additional resources section.

If we want to get into the difference between additional resources and next steps, I would separate that out into its own section instead of burying it here under the link sentences

Reply:
What's a better term for "link sentences" here? I gathered he meant the title without extra text, but you're right that it would be more clear to avoid saying "sentence" at all - I wasn't sure what to suggest though

Reply:
In the contributing guide, "link and xref text" is used. So maybe just "text"? That way its clear that this shouldn't be a sentence.

Reply:
We must consider when piinting to links that exist outside of a document (another Red Hat product document). How best to handle this case? Add the document name in parentheses?

Example:

> For more information about Cryostat's default event templates, see [Event templates]() (Using Cryostat to manage a JFR recording).

Reply:
Best to go with this wording as threee variants exist according to our guidelines: 

![Screenshot from 2024-03-22 09-46-45](https://github.com/openshift/openshift-docs/assets/57954076/783d8c89-fbb2-44db-8423-a3d62522353f)

Reply:
The mod doc guidelines has a few variations to their Add.R examples. Here is a complete sentence used in one example:
 
![Screenshot from 2024-03-22 09-48-06](https://github.com/openshift/openshift-docs/assets/57954076/3764b7b8-049a-40ce-8496-9bbf1d3be66d)

I think we need to focus on what format is best for conssitency across OCP docs. OCP docs take on their own specific guidelines in certain instances if I'm correct?

Reply:
Thanks, @jeana-redhat . How about:

> When pointing to a link in a section that exists in another customer product document, use the `For more information on <topic>, see "<section_name>" in the _<document_name>_ documentation.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.

Reply:
I'm a bit lost at this point about which line we're discussing :sweat_smile:

Reply:
:smile: True that, Jeana. I need to detail a few more variances as I was thinking too narrowly. Here is the sentence under the lens:

>  When pointing to a link in a section that exists in another customer product document, use the `For more information on <_topic>, see "<section_name>" in _<document_name>_.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.